src/canvas/canvasRenderer.js

```diff
@@ -111,6 +111,13 @@ var canvasRenderer = function (arg) {
         features = layer.features(),
         i;
 
+    for (i = 0; i < features.length; i += 1) {
+      if (features[i]._delayRender()) {
```

Comment:
should it be  if (!features[i]._delayRender())? If delayRender is true which means you want to skip rendering or something I missed here?

Reply:
Calling `render()` reschedules the rendering to the next animation frame.  The `return` then stops it from rendering right now.

Reply:
ok, thanks perhaps adding a comment here might help but I am fine either way.